packages/blockchain-wallet-v4/src/network/api/experiments/index.ts

```diff
@@ -0,0 +1,13 @@
+import { Experiments } from './types'
+
+export default ({ authorizedGet, nabuUrl }) => {
+  const getExperiments = (): Experiments =>
```

Comment:
I did test all redux and redux saga bindings and everything seems to work fine. But please check my code!